src/igvfd/tests/fixtures/schemas/alignment_file.py

```diff
@@ -166,3 +166,15 @@ def alignment_file_v11(testapp, lab, award, principal_analysis_set, reference_fi
         'controlled_access': False
     }
     return item
+
+
+@pytest.fixture
+def alignment_file_v12(alignment_file_v11):
+    item = alignment_file_v11.copy()
```

Comment:
I'd suggest just copying the base fixture alignment_file, the versioned fixtures typically have deprecated properties/values. won't cause any problems here but it can in theory